src/components/topbar/topbar-global.tsx

```diff
@@ -120,7 +120,7 @@ const TopbarGlobal: FC<TopbarProps> = ({ domain: _domain }) => {
         <div className="topHalf">
           <div className="topLeft">
             <Link to={'/'} className="network">
-              0://
+              0:/ <span style={{ marginLeft: '-5px' }}>/</span>
```

Comment:
Would it be possible to avoid using inline styles?

Reply:
thanks for the feedback!
-fixed adbar background scaling
-took out inline styles
-merged with develop